core-web/libs/template-builder/src/lib/components/template-builder/models/models.ts

```diff
@@ -26,3 +26,5 @@ export interface DotGridStackNode extends GridStackNode {
 export interface DotTemplateBuilderState {
     items: DotGridStackWidget[];
 }
+
+export type WidgetType = 'col' | 'row';
```

Comment:
I think is better just to pass the icon, this is just a presentational component.